src/screens/VisualTests/BuildResults.tsx

```diff
@@ -26,22 +27,26 @@ import { StoryInfo } from "./StoryInfo";
 import { Warnings } from "./Warnings";
 
 interface BuildResultsProps {
+  branch: string;
   runningBuild: RunningBuildPayload;
   storyBuild: StoryBuildFieldsFragment;
   nextBuild: NextBuildFieldsFragment;
   switchToNextBuild?: () => void;
   startDevBuild: () => void;
+  canReview: boolean;
   isReviewing: boolean;
   onAccept: (testId: string, batch: ReviewTestBatch) => Promise<void>;
   onUnaccept: (testId: string) => Promise<void>;
   setAccessToken: (accessToken: string | null) => void;
 }
 
 export const BuildResults = ({
+  branch,
   runningBuild,
   nextBuild,
   switchToNextBuild,
   startDevBuild,
+  canReview,
```

Comment:
I think it would make it more obvious why this is different to `isReviewable`.

```suggestion
  userCanReview,
```

 Maybe we should call that `storyBuildIsReviewable` too?